### ISSUE TITLE
Add types for isValidAddress

### ISSUE THREAD

**Issue by kincaidoneil** [ripple/ripple-lib#1032]
When using ripple-lib in a TypeScript project with `strict` mode enabled, I get this error:

```
node_modules/ripple-lib/dist/npm/common/schema-validator.d.ts:1:32 - error TS7016: Could not find a declaration file for module 'ripple-address-codec'. '/Users/kincaid/ilp/settlement-xrp/node_modules/ripple-address-codec/src/index.js' implicitly has an 'any' type.
  Try `npm install @types/ripple-address-codec` if it exists or add a new declaration (.d.ts) file containing `declare module 'ripple-address-codec';`

1 import { isValidAddress } from 'ripple-address-codec';
                                 ~~~~~~~~~~~~~~~~~~~~~~
```

Adding the suggested type declaration file fixes it, but it'd be nice if that wasn't required and/or `isValidAddress` was strongly typed (right now it's just `any`).

**Comment by intelliot** [comment]
Thanks for reporting this, I agree it would be nice to update `ripple-address-codec` to provide types. But is there a reason that you're importing `isValidAddress` from `ripple-address-codec`?

The same function is exposed through `ripple-lib`:
```
import { RippleAPI } from 'ripple-lib';
const api = new RippleAPI();
api.isValidAddress('rKcYDAqzwD8ioXBfzC1GMzSaEkVnQna7zr') // true
```

It's still `any`, but this way there should be no need to add a new declaration.

**Comment by kincaidoneil** [comment]
I'm not using `isValidAddress` at all, just importing ripple-lib. Here's a reproducible example: https://github.com/kincaidoneil/ripple-lib-test

**Comment by intelliot** [comment]
Ah, I see. Other projects I've seen use `"skipLibCheck": true` (in `tsconfig.json`).

**Comment by intelliot** [comment]
Fixed in ripple-lib 1.3.2
